handlers/handlers.go

```diff
@@ -72,15 +72,24 @@ func GetAreaViewHandler(w http.ResponseWriter, req *http.Request, ctx context.Co
 	}()
 	go func() {
 		defer wg.Done()
-		ancestorData, ancestorErr = c.AreaApi.GetAncestors(areaID)
+		// GetAncestors(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, areaID, acceptLang string) (ancestors Ancestors, err error)
+		ancestorData, ancestorErr = c.AreaApi.GetAncestors(ctx, accessToken, "", collectionID, areaID, acceptedLang)
 		if ancestorErr != nil {
 			log.Error(ctx, "fetching ancestor data", err)
 			return
 		}
 	}()
 	wg.Wait()
-	//  View logic
 	basePage := c.Render.NewBasePageModel()
+	if err != nil {
+		// No areaData - Could be invalid geocode
+		setStatusCode(req, w, err)
+		title := "404 page not found!"
+		description := "The requested page was not found"
```

Comment:
We have standard wording (which should really be in the dp-renderer locale files if you are up for it to prevent everyone having this issue in the future). An example of the wording can be found here: https://github.com/ONSdigital/dp-frontend-router/blob/ca043e4baa288361b6caeaa40680870fe834f909/middleware/serverError/handler.go#L30.